web/web/models/alert.py

```diff
@@ -1,28 +1,235 @@
-from marshmallow import Schema, fields
-#from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
-# from sqlalchemy.types import TIMESTAMP
-# from web.database import (
-#     db,
-#     Model,
-#     Column,
-#     SurrogatePK,
-#     relationship,
-#     reference_col,
-# )
-
-
-# class Alert(Model):
-#     __tablename__ = 'alerts'
-
-#     def __repr__(self):
-#         return f'<Alert>'
-
-
-class AlertSchema(Schema):
-    date = fields.Str()
-    time = fields.Str()
-    alert_type = fields.Str()
-    description = fields.Str()
-    status = fields.Str()
-    assigned_to = fields.Str()
-    resolution = fields.Str()
+import enum
+from sqlalchemy.types import TIMESTAMP
+from sqlalchemy import event, text, func
+from marshmallow import fields, ValidationError
+from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
+
+from web.emails.send_emails import send_email
+from web.models.notification import Notification
+from web.models.alert_type import AlertType
+from web.database import (
+    db,
+    Model,
+    Column,
+    SurrogatePK,
+    relationship,
+    reference_col,
+)
+
+class Status(enum.Enum):
+    OPEN = 'open'
+    PENDING = 'pending'
+    RESOLVED = 'resolved'
+
+    @staticmethod
+    def check_value(str_value):
+        '''Takes in string value, returns False if it isn't an accepted enum value, else returns enum type.'''
+
+        for status in Status:
+            if status.value == str_value:
+                return status
+
+        return False
+
+class AssignedToOptions(enum.Enum):
```

Comment:
I thought `assignedto` was simply a reference to the User model. We shouldn't need this enum.

Reply:
These specific enum types for AssignedToOptions do not exist on user or roles, such as operator 1, field crew etc. I'm not sure where the connection should be on user. unless there is some logic to differentiate a user that is "operator 1" or an additional column in RoleType for operator 1 vs field crew. Let me know which direction I should take, Thanks!

Reply:
I think operator 1, etc... were just examples. I think it should just be User.email, or User.last_name + User.first_name